docs/use/ble.md

```diff
@@ -160,11 +201,83 @@ you can also accept all the devices by the following command:
 
 The default value is set into config_BT.h
 
+## ADVANCED: Setting up an external decoder
+
+This advanced option is used to publish raw radio frames on a specific topic to be decoded by an external decoder instead of the integrated one.
+
+To enable external decoder:
+
+`mosquitto_pub -t home/OpenMQTTGateway/commands/MQTTtoBT/config -m '{"extDecoderEnable":true}'`
+
+To change the default external decoder topic to "undecoded":
+
+`mosquitto_pub -t home/OpenMQTTGateway/commands/MQTTtoBT/config -m '{"extDecoderTopic":"undecoded"}'`
+
+## ADVANCED: Filtering out connectable devices
+
+[With OpenHAB integration](../integrate/openhab2.md), this configuration is highly recommended, otherwise you may encounter incomplete data.
+
+If you want to enable this feature:
+
+`mosquitto_pub -t home/OpenMQTTGateway/commands/MQTTtoBT/config -m '{"filterConnectable":true}'`
+
+## ADVANCED: Publishing known service data
+
+If you want to enable this feature:
+
+`mosquitto_pub -t home/OpenMQTTGateway/commands/MQTTtoBT/config -m '{"pubKnownServiceData":true}'`
+
+## ADVANCED: Publishing unknown service data
+
+If you want to change the default behaviour, in case you are having too heavy service data:
+
+`mosquitto_pub -t home/OpenMQTTGateway/commands/MQTTtoBT/config -m '{"pubUnknownServiceData":false}'`
+
+## ADVANCED: Publishing known manufacturer's data
+
+If you want to change the default behaviour:
+
+`mosquitto_pub -t home/OpenMQTTGateway/commands/MQTTtoBT/config -m '{"pubKnownManufData":true}'`
+
+## ADVANCED: Publishing unknown manufacturer's data
+
+If you want to change the default behaviour, in case you are having too heavy service data:
+
+`mosquitto_pub -t home/OpenMQTTGateway/commands/MQTTtoBT/config -m '{"pubUnknownManufData":false}'`
+
+## ADVANCED: Publishing the service UUID data
+
+If you want to change the default behaviour:
+
+`mosquitto_pub -t home/OpenMQTTGateway/commands/MQTTtoBT/config -m '{"pubServiceDataUUID":true}'`
+
+## Store BLE configuration into the gateway (only with ESP32 boards)
+
+Open MQTT Gateway has the capability to save the current configuration and reload it at startup.
```

Comment:
Minor, could you just write OpenMQTTGateway in one word

Reply:
Sure, I'll update the PR

Reply:
we will fix it later on

Reply:
Sorry, I had a quite intense few days, I'll update that with the doc update at the same time as BL/WL